src/index.js

```diff
@@ -75,9 +77,19 @@ const IN_CALL_BUTTONS_ARRAY = [
         const deleteButtonElement = document.getElementById("delete-button")
         const dialButtonPadElement = document.getElementById("dial-button-pad")
         const callDeleteWrapperElement = document.getElementById("call-delete-wrapper")
+        const headerElement = document.querySelector("header")
```

Comment:
All these `document.getElementById` should be defined in GLOBAL scope if element they target is defined in `.html` file